README.md

```diff
@@ -1,85 +1,100 @@
+Although optional we recommend to install jenv to manage different Java versions, to do that run: `brew install jenv`
 
-- Set String generator = "federator1";
+Add jenv to your path as it says on the output of jenv brew installation.
 
-- Run the Class to generate a private key
+Download [oracle-jdk8](https://www.oracle.com/ar/java/technologies/javase/javase8-archive-downloads.html) and install it.
 
-- Create a file with only the private key as the content
 
-- Add permission to the file by running **chmod 400 reg1.key**
+**Note:** *if you have an Apple Silicon chip you may get a notification saying that is not recommended to install a file that is for Intel chips, just ignore it and keep going with the installation.*
 
-Optionally you can generate three files, one per each signer to be used (BTC/RSK/MST)
 
-2. Create the powpeg node config file.
+Add the just downloaded jdk to the jenv versions, run: `jenv add /Library/Java/JavaVirtualMachines/<ORACLE-JDK8-DIRECTORY>/Contents/Home/`
 
-This file include at least the bitcoin node connection information and the signers' configuration.
+If you are using another java version in other projects, run `jenv local 1.8.0.352`. Otherwise you can set it globally by running: `jenv global 1.8.0.352`
 
-Down below there is an example you can use (**(For Local test / regtest only)**)
+To check that it worked, open a new terminal instance and run `java -version` command. It should output something like 
 
 ```
-federator {
-    enabled = true
-    
-    signers {
-       BTC {
-          type = "keyFile"
-          path = "A/PATH/TO/YOUR/BTC-KEY.key" # This should point to your generated key file
-       }
-       RSK {
-          type = "keyFile"
-          path = "A/PATH/TO/YOUR/RSK-KEY.key" # This should point to your generated key file
-       }
-       MST {
-          type = "keyFile"
-          path = "A/PATH/TO/YOUR/MST-KEY.key" # This should point to your generated key file
-       }
-    }
-    # peers for the bitcoin network
-    bitcoinPeerAddresses = [
-        "127.0.0.1:18444" #bitcoind p2p port.
-    ]
-}
+openjdk version "1.8.0_352"
+OpenJDK Runtime Environment
+OpenJDK 64-Bit Server VM 
 ```
-**Note: You can use the same key file for BTC, RSK, and MST**
 
-**Important:**
-When setting up the Bitcoind Node host:port for the powpeg-node you MUST use the p2p port.
-The powpeg-node will “connect” even if you setup the RPC port, but the connection won’t go through and the powpeg-node won’t start.
+*If not, try closing the terminal and open a new one cause sometimes it stays cached.*
+
+### Bitcoind
 
-### Fetch the code!
+To install version `0.18.0`
 
 ```bash
-git clone https://github.com/rsksmart/powpeg-node
-cd powpeg-node
+curl -O https://bitcoin.org/bin/bitcoin-core-0.18.0/bitcoin-0.18.0-osx64.tar.gz
+tar -zxf bitcoin-0.18.0-osx64.tar.gz
+sudo mkdir -p /usr/local/bin
+sudo cp bitcoin-0.18.0/bin/bitcoin* /usr/local/bin/.
+rm -rf bitcoin-0.18.0
```

Comment:
This command is not necessary. It is just deleting the folder you just unzip.

Reply:
ok! ill delete the rm -rf line